ci-operator/step-registry/gcp-hcp/terraform-plan/gcp-hcp-terraform-plan-commands.sh

```
@@ -0,0 +1,23 @@
+
+set -euo pipefail
+
+if [[ ! -f "${SHARED_DIR}/wif-cred.json" ]]; then
+  echo "ERROR: ${SHARED_DIR}/wif-cred.json not found — hypershift-gcp-wif-auth step must run first"
+  exit 1
+fi
+
+gcloud auth login --cred-file="${SHARED_DIR}/wif-cred.json"
+
+CI_FOLDER_ID="$(<"${CLUSTER_PROFILE_DIR}/ci-folder-id")"
+BILLING_ACCOUNT_ID="$(<"${CLUSTER_PROFILE_DIR}/billing-account-id")"
```

Shellcheck (0.11.0)

[warning] BILLING_ACCOUNT_ID appears unused. Verify use (or export if used externally).

(SC2034)